feature/session/src/test/java/io/github/droidkaigi/confsched2019/model/SessionContentsTest.kt

```diff
@@ -0,0 +1,126 @@
+package io.github.droidkaigi.confsched2019.model
+
+import io.github.droidkaigi.confsched2019.dummyServiceSessionData
+import io.github.droidkaigi.confsched2019.dummySpeakerData
+import io.github.droidkaigi.confsched2019.dummySpeechSessionData
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class SessionContentsTest {
+
+    // Session TestCase
+
+    @Test fun search_WhenSpeechSessionFilteredAndTitleEn() =
```

Comment:
Can you do it with `Parameterized`? It is a little difficult (I can't understand this test without 2 definition dump 😭 )
https://github.com/DroidKaigi/conference-app-2019/blob/2c9030940154f367d36d10f0ecd6911abe4879d3/model/src/jvmTest/kotlin/io/github/droidkaigi/confsched2019/model/FiltersTest.kt#L25

Reply:
Let's try a test case using Parameterized.